Fake.sln

```diff
@@ -1,4 +1,5 @@
-﻿Microsoft Visual Studio Solution File, Format Version 12.00
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Can we remove this change?

Reply:
Yep, just autogenerated msvc stuff, didn't realize that was added in